src/devTools.js

```diff
@@ -187,12 +211,36 @@ class DevToolsEnhancer {
     }
   };
 
+  startWrapper = async () => {
+    if (this.started || (this.socket && this.socket.getState() === this.socket.CONNECTING)) {
+      return;
+    }
+
+    if (!this.socketOptions.port) {
+      // no port provided - we should throw!
+      throw new Error('no port provided');
+    }
+    if (this.socketOptions.hostname) {
+      // hostname provided - don't look for it
+      this.start();
+      return;
+    }
+    // obtain the hostname
+    try {
+      this.socketHostname = await this.hostnamePromise;
```

Comment:
Why don’t we just assign to `this.socketOptions.hostname` directly?

Reply:
right, seems like `socketHostname` is no longer needed and can be removed, thanks.